Update systel.edf.cfg for EasyBuild: rename S10.gfortran to easybuild,
remove optional MED/AED/GOTM dependencies, extend MUMPS linking, use $F90FLAGS,
add -fallow-invalid-boz, and disable hyperthreading in MPI commands.
Author: Cintia Willemyns (Vrije Universiteit Brussel)
--- telemac-mascaret-v9.0.0.orig/configs/systel.edf.cfg	2025-10-27 15:26:19.000000000 +0100
+++ telemac-mascaret-v9.0.0/configs/systel.edf.cfg	2025-11-06 11:06:27.000000000 +0100
@@ -2,7 +2,7 @@
 # ____/ TELEMAC Project Definitions /______________________________/
 #
 [Configurations]
-configs: S10.gfortran.dyn S10.gfortran.debug
+configs: easybuild.dyn easybuild.debug
          S10.nag.dyn S10.nag.debug
          S11.gfortran.dyn S11.gfortran.debug
          S11.nag.dyn S11.nag.debug
@@ -48,31 +48,32 @@
 #
 # MED info
 #
-incs_med: -I$MEDHOME/include
-flags_med: -DHAVE_MED
-libs_med: -lm -L$MEDHOME/lib -lmed
+incs_med:
+flags_med: 
+libs_med: 
 #
 # MUMPS info
 #
 incs_mumps: -I$MUMPSHOME/include
 flags_mumps: -DHAVE_MUMPS
-libs_mumps: -L$MUMPSHOME/lib -ldmumps
-#
+libs_mumps_core: -L$MUMPSHOME/lib -ldmumps -lmumps_common -lpord
+libs_mumps_extra: -L$SCALAPACKHOME/lib -lscalapack -lflexiblas -lgfortran -L$SCOTCHHOME/lib -lptscotch -lptscotcherr -lesmumps -lscotch -lscotcherr
+libs_mumps: [libs_mumps_core] [libs_mumps_extra]
 # AED info
 #
-incs_aed: -I$AEDHOME/include
-flags_aed: -DHAVE_AED2
-libs_aed: -L$AEDHOME/lib -laed2
+incs_aed: 
+flags_aed: 
+libs_aed: 
 #
 # GOTM info
 #
-incs_gotm: -I $GOTMHOME/include
-flags_gotm: -DHAVE_GOTM
-libs_gotm: -L$GOTMHOME/lib -lturbulence -lutil
+incs_gotm:
+flags_gotm: 
+libs_gotm: 
 #
 # GFortran info
 #
-fflags_gfo: -cpp -fconvert=big-endian -frecord-marker=4 -DHAVE_I16
+fflags_gfo: -fallow-invalid-boz -cpp -fconvert=big-endian -frecord-marker=4 -DHAVE_I16
 fflags_debug_gfo: -g -Wall -fcheck=all -fbacktrace -fbounds-check -finit-integer=-1 -finit-real=nan -ffpe-trap=invalid,zero,overflow
 #
 # Intel Fortran compilation flags
@@ -129,7 +130,7 @@
 #
 # GFortran
 #
-[S10.gfortran.dyn]
+[easybuild.dyn]
 brief: Scibian 10 dynamic build using GFortran 8.3.0 and Open MPI.
 options: api
 #
@@ -137,7 +138,7 @@
 pyd_fcompiler: gnu95
 sfx_lib:    .so
 #
-obj_flags: -O2 -fPIC [fflags_gfo] [flags_mpi] [flags_mumps] [flags_med] [flags_aed] [flags_gotm]
+obj_flags: $F90FLAGS -fPIC [fflags_gfo] [flags_mpi] [flags_mumps] [flags_med] [flags_aed] [flags_gotm]
 lib_flags: -fPIC -shared [fflags_gfo]
 #
 exe_flags: -fPIC [fflags_gfo]
@@ -145,9 +146,9 @@
 cmd_lib:    [fc] [lib_flags] -o <libname> <objs>
 cflags: -fPIC
 #
-mpi_cmdexec: mpirun --use-hwthread-cpus -np <ncsize> <exename>
+mpi_cmdexec: mpirun -np <ncsize> <exename>
 #
-[S10.gfortran.debug]
+[easybuild.debug]
 brief: Scibian 10 dynamic build using GFortran 8.3.0 and Open MPI, in debug mode.
 options: api
 #
@@ -164,7 +165,7 @@
 #
 cmd_lib: [fc] [lib_flags] -o <libname> <objs>
 #
-mpi_cmdexec: mpirun --use-hwthread-cpus -np <ncsize> <exename>
+mpi_cmdexec: mpirun -np <ncsize> <exename>
 #
 # NAG
 #
